samples-aspnetcore-2x/self-hosted-login/README.md

```diff
@@ -21,13 +21,15 @@ Run the example with your preferred tool and write down the port of your web app
 
 > **NOTE:** This sample is using ASP.NET Core 2.2 which enforces HTTPS. This is a recommended practice for web applications. Check out [Enforce HTTPS in ASP.NET Core] for more details.
 
+> Because of recent changes in Set-Cookie behavior (SameSite) this code will only work properly if it's configured to use https.
```

Comment:
```suggestion
> Because of recent changes in Set-Cookie behavior (SameSite) this code will only work properly if it's configured to use https. Check out [Work with SameSite cookies in ASP.NET Core](https://docs.microsoft.com/en-us/aspnet/core/security/samesite?view=aspnetcore-3.1) for more details.
```

Reply:
Can you please this change (MS SameSite docs) in the other READMEs?

Reply:
Will do, but don't you think it's better to put here the link Chrome shows in the Devtools warning message? (https://web.dev/samesite-cookies-explained/)

Reply:
Good idea! I think we can use both. I'll update the suggestions but feel free to adjust it if you find a better way.

Reply:
I changed the text as you've suggested.